proxy/src/lib.rs

```diff
@@ -186,6 +187,14 @@ where
             "serving Prometheus metrics on {:?}",
             metrics_listener.local_addr(),
         );
+        info!(
+            "private listener disables protocol detection on ports {:?}",
```

Comment:
I suggest "Protocol detection disabled for outgoing destination ports {:?}":

1. "private listener" is not relevant to users.
2. "outgoing" indicates that it's for outgoing traffic.
3. "destination" indicates it's looking at the destination port, not the source port.